test/conversions.jl

```diff
@@ -1,79 +1,74 @@
 module TestConversions
     using Base.Test
-    using DataTables
+    using DataTables, Nulls
     using DataStructures: OrderedDict, SortedDict
 
     dt = DataTable()
     dt[:A] = 1:5
     dt[:B] = [:A, :B, :C, :D, :E]
     @test isa(convert(Array, dt), Matrix{Any})
-    @test convert(Array, dt) == convert(Array, convert(NullableArray, dt))
     @test isa(convert(Array{Any}, dt), Matrix{Any})
 
     dt = DataTable()
     dt[:A] = 1:5
     dt[:B] = 1.0:5.0
-    # Fails on Julia 0.4 since promote_type(Nullable{Int}, Nullable{Float64}) gives Nullable{T}
-    if VERSION >= v"0.5.0-dev"
-        @test isa(convert(Array, dt), Matrix{Float64})
-    end
-    @test convert(Array, dt) == convert(Array, convert(NullableArray, dt))
+    @test isa(convert(Array, dt), Matrix{Float64})
     @test isa(convert(Array{Any}, dt), Matrix{Any})
     @test isa(convert(Array{Float64}, dt), Matrix{Float64})
 
     dt = DataTable()
-    dt[:A] = NullableArray(1.0:5.0)
-    dt[:B] = NullableArray(1.0:5.0)
+    dt[:A] = Vector(1.0:5.0)
+    dt[:B] = Vector(1.0:5.0)
     a = convert(Array, dt)
     aa = convert(Array{Any}, dt)
     ai = convert(Array{Int}, dt)
     @test isa(a, Matrix{Float64})
-    @test a == convert(Array, convert(NullableArray, dt))
     @test a == convert(Matrix, dt)
     @test isa(aa, Matrix{Any})
     @test aa == convert(Matrix{Any}, dt)
     @test isa(ai, Matrix{Int})
     @test ai == convert(Matrix{Int}, dt)
 
-    dt[1,1] = Nullable()
-    @test_throws ErrorException convert(Array, dt)
-    na = convert(NullableArray, dt)
-    naa = convert(NullableArray{Any}, dt)
-    nai = convert(NullableArray{Int}, dt)
-    @test isa(na, NullableMatrix{Float64})
-    @test isequal(na, convert(NullableMatrix, dt))
-    @test isa(naa, NullableMatrix{Any})
-    @test isequal(naa, convert(NullableMatrix{Any}, dt))
-    @test isa(nai, NullableMatrix{Int})
-    @test isequal(nai, convert(NullableMatrix{Int}, dt))
+    @test_throws MethodError dt[1,1] = null
+    dt[:A] = Vector{?Float64}(1.0:5.0)
+    dt[1, 1] = null
+    na = convert(Array{?Float64}, dt)
```

Comment:
Should also test the conversion to `Array` without specifying the element type (as before).